src/Files.App/App.xaml.cs

```diff
@@ -356,6 +357,8 @@ public static void SaveSessionTabs()
 				}
 			})
 			.ToList();
+			MainPageViewModel mainPageViewModel = Ioc.Default.GetRequiredService<MainPageViewModel>();
+			mainPageViewModel.RemoveThisInstanceTabs();
```

Comment:
App.xaml.cs isn't supposed to access mainPageViewModel

Reply:
I changed it. App.xaml.cs now will directly access the SharedMemory and won't access MainPageViewModel.